Core/include/Acts/EventData/SingleFreeTrackParameters.hpp

```diff
@@ -167,44 +165,15 @@ class SingleFreeTrackParameters {
     m_oParameters.setParameter<kIndex>(newValue);
   }
 
-  /// @brief Print information to output stream
-  ///
-  /// @param [in, out] sl The output stream
+  /// Print information to the output stream.
   ///
-  /// @return The modified output stream object @p sl
-  std::ostream& print(std::ostream& sl) const {
-    // Set stream output format
-    auto old_precision = sl.precision(7);
-    auto old_flags = sl.setf(std::ios::fixed);
-
-    // Fill stream with content
-    sl << " * FreeTrackParameters: ";
-    sl << parameters().transpose() << std::endl;
-    sl << " * charge: " << charge() << std::endl;
-    if (covariance().has_value()) {
-      sl << " * covariance matrix:\n" << *covariance() << std::endl;
-    } else {
-      sl << " * no covariance matrix stored" << std::endl;
-    }
-
-    // Reset stream format
-    sl.precision(old_precision);
-    sl.setf(old_flags);
-
-    return sl;
-  }
-
-  /// @brief Output stream operator
-  ///
-  /// Prints information about this object to the output stream
-  /// @param [in, out] out The output stream
-  /// @param [in] sfp The object that will be printed
-  ///
-  /// @return Modified output stream object
-  friend std::ostream& operator<<(std::ostream& out,
-                                  const SingleFreeTrackParameters& sfp) {
-    sfp.print(out);
-    return out;
+  /// @param os The output stream
+  /// @return The modified output stream
+  std::ostream& print(std::ostream& os) const {
```

Comment:
The free parameters class had a public `.print(...)` method before, but the bound parameters did not. I'll check if the public `.print(...)` is actually used and remove it otherwise.

Reply:
Has been removed now.